### ISSUE TITLE
Package search doesn't match all package names

### ISSUE THREAD

**Issue by genodeftest** [fedora-infra/fedora-packages#215]
Steps to reproduce:
1. go to a.fp.o
2. search for `restorecond`

Expected result:
Searching for `restorecond` should match `policycoreutils-restorecond`

Actual result:
No search results

**Comment by ralphbean** [comment]
Agreed.  This is some kind of bug in the way we are constructing our xapian queries under the hood.
